backend/pkg/api/enums/enums.go

```diff
@@ -202,6 +202,20 @@ var ValidatorDuties = struct {
 	DutySlashed,
 }
 
+// ----------------
+// Chart Types
+
+type ChartType int
+
+const (
+	ChartDefault ChartType = iota
+	ChartRewards
+)
+
+func (d ChartType) Int() int {
```

Comment:
What is the int value that we pass is invalid?

Should we have a function that checks the valid number
```
func (d ChartType) IsValid() bool {
    return d >= ChartDefault && d <= ChartRewards
}
```

and maybe a function that stringify for debugging and logging purposes?
```
func (d ChartType) String() string {
    switch d {
    case ChartDefault:
        return "Default"
    case ChartRewards:
        return "Rewards"
    default:
        return "Unknown"
    }
}
```

Reply:
>What is the int value that we pass is invalid?

This should be covered here already: https://github.com/gobitfly/beaconchain/pull/1220/files#diff-7332d3a9f5362468b96f4180ca7683ea1dd73b6d13bdfe10f8b40b9c400869bfR15-R17

>and maybe a function that stringify for debugging and logging purposes?

It's meant as an internal type only, thought it's cleaner than passing strings or numbers. It doesn't depend on user params and isn't logged anywhere, so I think that's not needed.